Makefile

```diff
@@ -84,7 +85,10 @@ pkg-remove:
 pkg-list:
 	@$(GOPATH)/bin/govendor list
 
-install: gomake-all
+# Builds minio and installs it to $GOPATH/bin.
+install: build
+	@echo "Installing minio at $(GOPATH)/bin/minio"
+	@cp $(PWD)/minio $(GOPATH)/bin/minio
```

Comment:
We can print some friendly message like `Minio compiled successfully  'minio server export' to start.`